src/rpc/handlers/AccountNFTs.cpp

```diff
@@ -77,11 +77,19 @@ AccountNFTsHandler::process(AccountNFTsHandler::Input input, Context const& ctx)
     auto const pageKey =
         input.marker ? ripple::uint256{input.marker->c_str()} : ripple::keylet::nftpage_max(*accountID).key;
     auto const blob = sharedPtrBackend_->fetchLedgerObject(pageKey, lgrInfo.seq, ctx.yield);
+    auto const markerSet = input.marker.has_value();
 
-    if (!blob)
+    if (!blob) {
+        if (markerSet)
```

Comment:
Since markerSet is used only inside the if, we may move 80 into if branch, or just use input.marker.has_value() here